python-sdk/tutorials/deploy-with-prebuilt-docker-images/pytorch-prebuilt-docker-image.ipynb

```diff
@@ -0,0 +1,938 @@
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.06299,
+          "end_time": "2021-05-18T20:36:47.376128",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:47.313138",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from azureml.core.model import Model\n",
+        "\n",
+        "# Register the model\n",
+        "model = Model.register(\n",
+        "    model_name=\"pytorch-birds-model\",\n",
+        "    model_path=\"pytorch-birds/outputs/model.pt\",\n",
+        "    workspace=ws,\n",
+        ")"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "collapsed": true,
+        "gather": {
+          "logged": 1620920642194
+        },
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 2.978797,
+          "end_time": "2021-05-18T20:36:50.424619",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:47.445822",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Deploy model as web service\n",
+        "\n",
+        "Once you have your model, you can deploy the model on Azure. In this tutorial, we will deploy the model as a web service in [Azure Container Instances (ACI)](https://docs.microsoft.com/en-us/azure/container-instances/).\n",
+        "For more information on deploying models using Azure Machine Learning, refer [here](https://docs.microsoft.com/azure/machine-learning/service/how-to-deploy-and-where)."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.048629,
+          "end_time": "2021-05-18T20:36:50.604201",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:50.555572",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Write the requirements.txt\n",
+        "Write the `requirements.txt`\n",
+        "\n",
+        "> **Pin each package version:**\n",
+        "Even while prototyping, pin each package version in requirements.txt. \n",
+        "For example, use scipy == 1.2.3 instead of just scipy or even scipy > 1.2.3. \n",
+        "If you don't pin an exact version and scipy releases a new version, this can break your scoring script and cause failures during deployment and scaling."
```

Comment:
while true for production, for our examples we leave azureml-* packages unpinned to continually test on the latest version and ensure no breaking changes to our public-facing examples

Reply:
i am submitting pr with latest version
not pinning the version goes against the guidance